### PR TITLE
Issue 2 User View Entries

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-02-09 at 5 14 07 PM
Screen Shot 2021-02-09 at 5 14 20 PM

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -16,3 +17,28 @@ window.addEventListener('beforeunload', function (event) {
...
+  var $imgTag = $divColHalf1.appendChild(document.createElement('img'));
+  var $h3 = $divColHalf2.appendChild(document.createElement('h3'));
+  var $p = $divColHalf2.appendChild(document.createElement('p'));
+
+  $imgTag.setAttribute('src', entry.imageUrl);
+  $h3.textContent = entry.title;
+  $p.textContent = entry.notes;
+
+  return ($ul);
+}
```
Comment (codingcodymiller) [inline js/data.js]: This should be in `main.js`

File: js/data.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /* exported data */
 var previousDataJSON = localStorage.getItem('code-journal-local-storage');
+var $entries = document.querySelector('.row.entries');
```
Comment (codingcodymiller) [inline js/data.js]: This should be in `main.js`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! A couple quick changes:

**Comment by ameliatalkington** [comment]
Screen Shot 2021-02-09 at 6 18 55 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**